### ISSUE TITLE
Add ability to rotate sprites other than 90 degree angles?

### ISSUE THREAD

**Issue by TwoBitMachines** [piskelapp/piskel#1076]
Bug description

(this template is intended for bug reports, erase it when requesting features/enhancements)

### Steps to reproduce the bug
1. go to piskelapp.com
2. select a tool
3. ...

### Environment details
* operating system:
* browser (or offline application version):

### Sprite details
* number of frames:
* sprite resolution:
* session duration:

Feel free to include a screenshot of the bug, a .piskel file of the sprite or anything else that can help us investigate.

**Comment by PoolloverNathan** [comment]
As in 45°? This is hard to implement, as the pixels themselves are squares, and the desired outcome would depend on the person.

**Comment by TwoBitMachines** [comment]
Yes 45 or any angle in between really. I am aware that another sprite
editor sort of does this. So that's why I thought it was possible.

Thanks for your response.

**Comment by YansHJ** [comment]
> 

May I ask what the other one you're talking about is

**Comment by TwoBitMachines** [comment]
I believe it is aseprite.

**Comment by enykwest** [comment]
I submitted a pull request to address this: https://github.com/piskelapp/piskel/pull/1162
